packages/blade/src/components/Link/_decisions/decisions.md

```diff
@@ -0,0 +1,41 @@
+# Link Decisions <!-- omit in toc -->
+- [Design](#design)
+- [Link Component](#link-component)
+  - [Link API](#link-api)
+- [Component Breakdown](#component-breakdown)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=12699%3A147155) to the **Link** component
+
+
+## Link Component
+A Link component that can act as a `<button>` as well as an `<anchor>` element
+
+### Link API
+
+| Prop | Type | Required | Default | Description |
+|---|---|---|---|---|
+| variant | `anchor`, `button` | *No* | `anchor` | The variant of the Link to be rendered.<br><br>Note: `button` will look like an inline Link component (no spacing; can be rendered alongside other text) but will be rendered as a `button` element. |
+| children | `string` | When `icon` is passed: *No*<br>When `icon` is not passed: *Yes*   | undefined | The text to be rendered within the link. |
+| icon | `Icon` | When `children` is passed: *No*<br>When `children` is not passed: *Yes* | undefined | The Blade `Icon` component to be rendered within the link. |
+| iconPosition | `left`, `right` | *No* | `right` | The position of the rendered icon. |
+| isDisabled | `boolean` | *No* | `false` | Control whether the link is disabled or not.<br><br>Note: This prop is only valid for `button` variant. |
```

Comment:
[Yeah it can](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wG4AocmATzCTgGFcQIA7AGWFYGsAFHMAGc4AXjgBvcnDgA3FFGApWMAFwElaABbR8cAD4EARgFcYMNmSlxNRTAH41gmAtYBzCgF9KNOnE49+CCEAQVYtaFEJKzkFJVV1MO0oS2k2ABtqfz4BR2cud3IvH3oswKEAIVNzVkjJaRjFZTV8EzMLCmlgQQARLpRDNKQAEwc4QwgIQaVPb1oSrmyg4TEmEBYOBbLhADI4AApSgUFQ8Kh9P02jyrbWAEoKNDYnC55IvbAjtUOl29EAPii0iIMGMUBqrGMaTSM3IpAAPFlZPJGjARAAiDRJNFwLq9QT9QZDODpTKXILo7EAej+sIRCyRsWU6Na1WxuL6A2GxNYGW+YApcGpQA) we are already using a lot of conditional typescript props in BaseText, Button and other components.